modules/config/webapp.bicep

```diff
@@ -3,8 +3,8 @@ import { ConnectionStringDictionary } from '../utils/types.bicep'
 param appName string
 
 // az webapp list-runtimes --os-type linux
-@allowed(['DOTNETCORE|8.0', 'NODE|20-lts', 'PYTHON|3.12', 'DOCKER'])
-param appFramework string = 'DOTNETCORE|8.0'
+@allowed(['DOTNETCORE|10.0', 'NODE|20-lts', 'PYTHON|3.12', 'DOCKER'])
```

Comment:
Is it worth bumping (or adding) node to 24 (current LTS) while we're here?

(And possibly adding newer python, based on app service support)

Just considering the need for publishing bicep module versions each time we change.

Reply:
Updated